tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_container.ts

```diff
@@ -29,12 +30,17 @@ import {GroupBy} from '../../types';
   template: `
     <runs-group-menu-button-component
       [selectedGroupBy]="selectedGroupBy$ | async"
+      [showGroupByRegex]="showGroupByRegex$ | async"
       (onGroupByChange)="onGroupByChange($event)"
     ></runs-group-menu-button-component>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RunsGroupMenuButtonContainer {
+  showGroupByRegex$?: Observable<boolean> = this.store.select(
```

Comment:
Can the '?' be dropped, since this field is initialized and never set to null/undefined?

Reply:
ah you're right. thanks for catching this. removed